bazel/envoy_build_system.bzl

```diff
@@ -28,7 +28,7 @@ def envoy_copts(repository, test = False):
         # TCLAP command line parser needs this to support int64_t/uint64_t
         "@bazel_tools//tools/osx:darwin": ["-DHAVE_LONG_LONG"],
         "//conditions:default": [],
-    }) + envoy_select_hot_restart(["-DENVOY_HOT_RESTART"])
```

Comment:
Yeah, this is weird, this definition should have been pulled into the consuming project.